recursive-list-main/src/components/list/list.component.tsx

```diff
@@ -0,0 +1,74 @@
+import { Grid } from "@material-ui/core";
+import { File, Folder } from "components";
+import { useCallback } from "react";
+import { Route, Routes, useLocation } from "react-router-dom";
+import { getParamFromPathname } from "utils/params/params.util";
+import { useListStyles } from "./list.styles";
+
+export interface List {
```

Comment:
move to interface file

Reply:
Created separate file.interfaces.ts for every component